app/src/main/java/waiter/ClientConnection.java

```diff
@@ -0,0 +1,27 @@
+package waiter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.net.Socket;
+
+public class ClientConnection implements Connection {
+
+    private final Socket client;
+
+    ClientConnection(Socket client) {
```

Comment:
Can we call this socket instead of client?

Reply:
I renamed it to `clientSocket` in my name change commit I have, but I can change it to socket instead if that makes more sense

Reply:
I see. I find the name `clientSocket` confusing because my understanding is that socket exists on the server rather than on the client. Maybe it's more appropriate call a connection clientConnection. I hope that makes sense.